src/components/ebay-button/index.js

```diff
@@ -15,33 +15,35 @@ function getTemplateData(state, input) {
     const priority = input.priority || 'secondary';
     const size = input.size;
     const fluid = input.fluid;
-    let classes = ['btn'];
+    let variant = input.variant;
     const model = {};
     let tag;
+    let mainClass = 'btn';
 
-    if (priority === 'primary' || priority === 'secondary') {
-        classes.push(`btn--${priority}`);
+    if (href) {
+        variant = 'fake';
+        tag = 'a';
+        model.href = href;
+    } else {
+        tag = 'button';
     }
 
-    if (size === 'small' || size === 'large') {
-        classes.push(`btn--${size}`);
+    if (href || variant === 'expand' || variant === 'cta') {
```

Comment:
Why not have `variant === 'fake' || ...` and then you can remove the `variant = 'fake';` assignment you make?

Reply:
Ah, because you might not have the variant with an `href` version. Gotcha. Never mind.

Reply:
It's a little weird -- `fake` is a considered a variant in Skin terms, but we use `href` as a proxy for it here. I was hoping to avoid a user doing something like 
```marko
<ebay-button variant="fake"> <!-- invalid, no href -->
```

Reply:
Yup. As a developer, I can see how that would confuse things.